samples/bluetooth/fast_pair/locator_tag/sysbuild/CMakeLists.txt

```diff
@@ -4,4 +4,6 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-SB_CONFIG_BT_FAST_PAIR=y
+find_package(Sysbuild REQUIRED HINTS $ENV{ZEPHYR_BASE})
```

Comment:
We do not need to create the `sysbuild/CMakeLists.txt` file for now (still we can keep it since it would be used after we introduce bootloader support to the locator tag)

Reply:
I know, as i said during our discussion on teams, I'll leave it in this PR.